src/platform/site-wide/sass/merger.scss

```diff
@@ -506,6 +506,12 @@
   .va-sidebarnav {
     .left-side-nav-title {
       padding-bottom: 25px;
+      > .icon-small {
+        vertical-align: middle;
+        padding-left: initial;
+        margin-right: 5px;
+        font-size: 1.7rem;
```

Comment:
There's a slight font-size difference between the homepage icons and these sidebar icons, even though they're both in 32x32 circles.